src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -57,6 +58,10 @@ export type DetailsViewProps = EntityDetailsListSharedProps & {
   selectAllIsChecked?: boolean
   /** The text to show for selecting the latest version, if it can be selected. Default is "Always Latest Version" */
   latestVersionText?: string
+  /** This request object is only used to tell react-query to cancel fetching all children at once. */
+  getChildrenInfiniteRequestObject?: EntityChildrenRequest
+   /** The total number of entities that can be retrieved */
+  totalProgress?: number
```

Comment:
The previous name of this prop was fine--totalEntities or totalNumberOfEntities. That info is still relevant at this level. Just should be called `totalProgress` in the BlockingLoader